lib/elixir_jobs_web/templates/offer/_form.html.eex

```diff
@@ -30,7 +30,20 @@
 
     <fieldset>
       <div class="field-container">
+        <%= textarea(f, :summary,
+          rows: 3,
+          maxlength: 200,
+          id: "offer_summary",
+          class: class_with_error(f, :summary, "summary"),
+          placeholder: gettext("Offer summary (max 200 characters)")) %>
```

Comment:
I would put something more like `Summary (will be shown in listings, max 200 characters)`